.github/workflows/main.yml

```diff
@@ -188,8 +191,10 @@ jobs:
         cd valgrind-3.20.0
```

Comment:
Something that might speed up the job is remove the `v` from `tar xvf`, because that produces a lot of lines and I suspect it's not cheap to transfer thousands of lines.

Reply:
Removed